File: atcoder/src/B4_11365.cs
// https://www.acmicpc.net/problem/11365
// 00:00:00
#nullable disable

using System.Text;

class B4_11365
{
	/*
	static void Main()
	{
        Console.WriteLine("--- Started ---");
		Solution();
        Console.WriteLine("--- Finished ---");
	}
	*/

	static void Solution()
	{
		var sb = new StringBuilder();
		while (true)
		{
			var str = Console.ReadLine();
			if ("END".Equals(str)) break;
			char[] chars = str.ToCharArray();
			Array.Reverse(chars);
	        sb.Append(chars).Append('\n');
		}
		Console.WriteLine(sb);
	}
}

File: javac/src/B4_11365.java
//// https://www.acmicpc.net/problem/11365
import java.io.*;

public class B4_11365 {
    public static void main(String[] args) throws IOException {
        try (var br = new BufferedReader(new InputStreamReader(System.in))) {
            var res = new StringBuilder();
            while(true) {
                var inp = new StringBuilder(br.readLine());
                if ("END".equals(inp.toString())) break;
                res.append(inp.reverse()).append('\n');
            }
            System.out.println(res);
        }
    }
}
